charts/images.yaml

```diff
@@ -36,4 +36,4 @@ images:
 - name: calico-cpva
   sourceRepository: github.com/kubernetes-sigs/cluster-proportional-vertical-autoscaler
   repository: k8s.gcr.io/cpvpa-amd64
-  tag: v0.8.3
+  tag: v0.8.4
```

Comment:
I believe we have to change the repository to be k8s.gcr.io/cpa/cpvpa. The current repository k8s.gcr.io/cpa/cpvpa-amd64 does not seem to be multi-arch.